File: java/CampaignsExample.java
public class CampaignsExample {
    public static void main(String[] args) {
        try {
            MailproClient client = new MailproClient("YOUR_API_KEY");

            String campaignName = "Java Campaign";
            String subject = "Special Offer";
            String from = "marketing@example.com";

            String response = client.createCampaign(campaignName, subject, from);
            System.out.println("Create Campaign Response: " + response);

        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: php/CampaignsExample.php
<?php
require_once "MailproClient.php";

$client = new MailproClient("YOUR_API_KEY");

try {
    $response = $client->createCampaign(
        "New Campaign",
        "Special Offer",
        "<h1>Don't miss our offer!</h1><p>Get 50% off today.</p>",
        "LIST_ID_HERE"
    );
    print_r($response);
} catch (Exception $e) {
    echo "Error: " . $e->getMessage();
}
?>
